### PR TITLE
2 fix dynamic solution check

### PR DESCRIPTION
**Pull request by DmytroOrlov**

### CODE REVIEW

File: server/app/controllers/TaskSolver.scala [hunk 1]
```diff
@@ -67,5 +100,9 @@ object TaskSolver {
 
   def sbt(command: String): Try[Boolean] = Try(Seq("sbt", command).! == 0)
 
-  lazy val sbtInstalled = sbt("sbtVersion").isSuccess // no exception, so sbt is in the PATH
+  // no exception, so sbt is in the PATH
+  lazy val sbtInstalled = sbt("--version").isSuccess
```
Comment (novakov-alexey) [inline server/app/controllers/TaskSolver.scala]: We check that sbt installed by looking at sbtVersion or by looking at version of our application? --version return version of application:
[devgym] $ --version
[info] client/*:version
[info]  0.1-SNAPSHOT
[info] server/*:version
[info]  1.0-SNAPSHOT

File: server/app/controllers/TaskSolver.scala [hunk 2]
```diff
@@ -39,26 +44,54 @@ class TaskSolver @Inject()(executor: RuntimeSuiteExecutor, dao: Dao, val message
...
+      } catch {
+        case NonFatal(e) => Future.successful(error(e.getMessage))
+      },
+      Some(Line("Compiling..."))
+    )
+  }
+
+  def runtimeTaskStream = WebSocket.acceptWithActor[JsValue, JsValue] { req => out =>
+    SimpleWebSocketActor.props(out, (fromClient: JsValue) => Future.successful((try {
+        val suiteClass = "tasktest.SubArrayWithMaxSumTest" // (fromClient \ "suiteClass").as[String]
```
Comment (novakov-alexey) [inline server/app/controllers/TaskSolver.scala]: can we switch to Dynamic executor using solution text from client?

File: server/app/controllers/TaskSolver.scala (hunk 1 above)
Comment (DmytroOrlov) [inline server/app/controllers/TaskSolver.scala]: Please read the code, the equivalent is `$ sbt --version` in your termina

File: server/app/controllers/TaskSolver.scala (hunk 2 above)
Comment (DmytroOrlov) [inline server/app/controllers/TaskSolver.scala]: I can't understand the question, aren't we?
`val solution = (fromClient \ "solution").as[String]`

File: server/app/controllers/TaskSolver.scala (hunk 1 above)
Comment (novakov-alexey) [inline server/app/controllers/TaskSolver.scala]: $ sbt --version
[error] Not a valid command: version (similar: session)
[error] version
[error]        ^

Comment (DmytroOrlov) [inline server/app/controllers/TaskSolver.scala]: dorlov@dmytro13:~$ sbt --version
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=384m; support was removed in 8.0
sbt launcher version 0.13.7

File: server/app/controllers/TaskSolver.scala (hunk 2 above)
Comment (novakov-alexey) [inline server/app/controllers/TaskSolver.scala]: As I understand, it should work via service.ScalaDynamicRunner#apply, but it is still on service.ScalaRuntimeRunner#apply

Comment (DmytroOrlov) [inline server/app/controllers/TaskSolver.scala]: but how it is working, it a kind of magic? xD
`executor: RuntimeSuiteExecutor with DynamicSuiteExecutor`

File: server/app/controllers/TaskSolver.scala (hunk 1 above)
Comment (novakov-alexey) [inline server/app/controllers/TaskSolver.scala]: I have sbt version 0.13.9, maybe this key "--version" was removed in 13.7. Not sure

File: server/app/controllers/TaskSolver.scala (hunk 2 above)
Comment (novakov-alexey) [inline server/app/controllers/TaskSolver.scala]: // (fromClient \ "suiteClass").as[String]
why it is commented?

Comment (DmytroOrlov) [inline server/app/controllers/TaskSolver.scala]: because it's a hint not implemented yet

Comment (novakov-alexey) [inline server/app/controllers/TaskSolver.scala]: that is was initial question. Can we switch....
Ok. It is clear

**Pull request merged**